README.md

```diff
@@ -133,15 +147,12 @@ match result {
 - [Lucio Franco](https://github.com/LucioFranco)
 - [Fletcher Haynes](https://github.com/fhaynes)
 - [Timon Post](https://github.com/TimonPost)
-- [Justin LeFebvre](https://github.com/jstnlef)
```

Comment:
We should leave Justin in the credits I think?

Reply:
He still is, check out the lines above that.

Reply:
Weird diff 👍. All good though.